hawkular-wildfly-agent-itest-parent/hawkular-wildfly-agent-itest-util/src/main/java/org/hawkular/wildfly/agent/itest/util/AbstractITest.java

```diff
@@ -222,6 +222,31 @@ protected String getNodeAttribute(ModelControllerClient mcc, ModelNode addressAc
         return actualAttributeValue;
     }
 
+    protected void writeNodeAttribute(ModelControllerClient mcc, ModelNode addressActual, String attributeName,
+            String value) {
+        OperationBuilder.writeAttribute()
+                .address(addressActual)
+                .attribute(attributeName, value)
+                .execute(mcc)
+                .assertSuccess();
+    }
+
+    protected void writeNodeAttribute(ModelControllerClient mcc, ModelNode addressActual, String attributeName,
+            Integer value) {
+        OperationBuilder.writeAttribute()
+                .address(addressActual)
+                .attribute(attributeName, value)
+                .execute(mcc)
+                .assertSuccess();
+    }
+
+    protected void restartAgent() {
```

Comment:
do you still need this restartAgent() method? Get rid of it if not needed anymore.